src/Form/ApsisMailSettings.php

```diff
@@ -154,7 +170,19 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         $form['demographic_data']['demographic_data'][$key]['required'] = [
           '#type' => 'checkbox',
           '#default_value' => !empty($demographic_data[$key]) ? $demographic_data[$key]['required'] : '',
-          '#disabled' => (count($alternatives) > 1 || !$alternatives) ? FALSE : TRUE,
+          '#disabled' => (count($alternatives) > 2 || !$alternatives) ? FALSE : TRUE,
```

Comment:
Der bliver refereret til de her alternativer som 1 og 2. Er det noget man kunne lave om til konstanter. Jeg kan ihvertfald som læser ikke umiddelbart forstå hvorfor der er to og tilsyneladende kun to alternativer.